python/dynamodb-lambda/README.md

```diff
@@ -13,7 +13,9 @@ This project is set up like a standard Python project.  The initialization
 process also creates a virtualenv within this project, stored under the .env
 directory.  To create the virtualenv it assumes that there is a `python3`
 (or `python` for Windows) executable in your path with access to the `venv`
-package. If for any reason the automatic creation of the virtualenv fails,
+package. It requires you to use Pythn 3.6 as Lambda runtime is set to Python 3.6. 
```

Comment:
Typo for Python.  Maybe update this to say you may have to match the python version that is specified in the dynamodb_lambda/dynamodb_lambda_stack.py file for your virtualenv.

We should get all those runtime references updated at some point.